packages/core/src/sso/SamlSsoConnector/index.ts

```diff
@@ -30,11 +31,8 @@ export class SamlSsoConnector extends SamlConnector implements SingleSignOn {
   ) {
     const parseConfigResult = samlConnectorConfigGuard.safeParse(data.config);
 
-    if (!parseConfigResult.success) {
-      throw new ConnectorError(ConnectorErrorCodes.InvalidConfig, parseConfigResult.error);
-    }
-
-    super(parseConfigResult.data, tenantId, data.id);
+    // Fallback to undefined if config is invalid
+    super(tenantId, data.id, conditional(parseConfigResult.success && parseConfigResult.data));
```

Comment:
Should not block the SAML instance from being created, if no proper idpConfig is provided. We rely on the instance to retrieve SP `metadata`, which does not rely on the `idpConfig`

Reply:
This comment is clear, why not add the comment to the code file.

Reply:
Updated